package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "flojoy-cloud",
   "scripts": {
-    "build": "turbo run build",
+    "fe:build": "turbo run build --filter={./apps/frontend}",
```

Comment:
what is the reason for filter here?

Reply:
@itsjoeoui To specify workspace location where to run build command from. Are you able to build frontend from root without specifying this option?

Reply:
Yep it works for me without any tweaks

Reply:
It worked for me too now :) Thanks. I'm reverting...